frontend/components/redeem/Receipt.tsx

```diff
@@ -29,12 +28,18 @@ const Receipt = ({ charity, primaryDonor, primaryDonorAmount, secondaryDonorAmou
           Donation Amount
         </Typography>
 
-        <ReceiptItem
-          imageBaseUrl={primaryDonor.imageBase64}
-          text={`$${primaryDonorAmount} from ${primaryDonor.name}`}
-        />
+        {couponSponsorship ? (
+          <>
+            <ReceiptItem
+              imageBaseUrl={couponSponsorship.primaryDonor.imageBase64}
+              text={`$${couponSponsorship.primaryDonorAmount} from ${couponSponsorship.primaryDonor.name}`}
+            />
 
-        <ReceiptItem text={`$${secondaryDonorAmount} from you`} />
+            <ReceiptItem text={`$${secondaryDonorAmount} from you`} />
+          </>
+        ) : (
+          <Typography variant="h4">${secondaryDonorAmount}</Typography>
```

Comment:
Is there a reason why we're not using ReceiptItem?

Reply:
<img width="585" alt="image" src="https://user-images.githubusercontent.com/75519136/197529493-00513c57-279f-46c2-b5fa-b25fb4be73a0.png">

I felt like this looked weird with just one avatar icon. It makes sense for redemption with the logo of the primary donor (and I know that this icon is me), but without the primary donor logo and just this avatar, I think it's confusing for the user

Reply:
@sivayogasubramanian said the ReceiptItem looks better, will use the ReceiptItem

Reply:
![image](https://user-images.githubusercontent.com/75519136/197536925-57289702-71ff-476c-b1ca-803405576b4e.png)